src/core/annotation.js

```diff
@@ -1596,11 +1597,13 @@ class ButtonWidgetAnnotation extends WidgetAnnotation {
   }
 
   _processPushButton(params) {
-    if (!params.dict.has("A")) {
+    if (!params.dict.has("A") && !params.dict.has("TU")) {
       warn("Push buttons without action dictionaries are not supported");
       return;
     }
 
+    this.data.isTooltipOnly = !params.dict.has("A");
```

Comment:
We cannot really test the display side of this patch, but we can unit test the core part, especially since we don't have unit tests for push buttons yet (we do have them for the other types of button widget annotations). Let's add two unit tests below https://github.com/mozilla/pdf.js/blob/master/test/unit/annotation_spec.js#L2319:

`it("should handle push buttons"` -> simple unit test to test rendering a regular push button, so without `TU` and with `A`
`it("should handle push buttons that act as a tooltip only"` -> same, but with `TU` and without `A`.